routes/api.php

```diff
@@ -14,6 +18,24 @@
 |
 */
 
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
+Route::post('/tokens/authenticate', function (LoginRequest $request) {
+    $request->authenticate();
+
+    $user = User::where('email', $request->get('email'))->first();
```

Comment:
When you have custom form request class `LoginRequest`, the proper way of accessing values is:

```php
$validated = $request->validated();
$email = $validated['email'];
```

or even better

```php
$email = $request->validated('email');
```